ccf_transaction_processor/Makefile

```diff
@@ -0,0 +1,136 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# SCRIPTDIR defaults to the current directory
+# SRCDIR defaults to the parent directory
+# DSTDIR defaults to _dev under the current directory
+# Both can be overridden with environment variables
+
+ifndef PDO_INSTALL_ROOT
+$(error Incomplete configuration, PDO_INSTALL_ROOT is not defined)
+endif
+
+ifndef SGX_MODE
+$(error Incomplete configuration, SGX_MODE is not defined)
+endif
+
+ifdef NINJA_OPTION
+NINJA_J_OPTION = $(NINJA_OPTION)
+else
+NINJA_J_OPTION = 4
+endif
```

Comment:
You could have written that way more compact as NINJA_J_OPTION ?= 4.

More confusing though, why do you expect NINJA_OPTION and then define NINJA_J_OPTION? If you really want to allow only override of -j, then you should also make the external variable NINJA_J_OPTION. Personally, i would have preferred the more general NINJA_OPTIONS i had in the patch i've sent you: more versatile. Also, why are you passing it only to ninja but not, say, ninja install?

Reply:
yes, thank you, (like we discussed), I missed the patch, I am applying the patch now. Thanks again.